src/snowflake/cli/api/utils/definition_rendering.py

```diff
@@ -386,3 +386,10 @@ def on_cycle_action(node: Node[TemplateVar]):
         default_env=project_context[CONTEXT_KEY].get("env"), override_env=override_env
     )
     return ProjectProperties(project_definition, project_context)
+
+
+def raw_project_properties(definition: Definition) -> ProjectProperties:
+    """
+    Returns the raw project definition data without any templating.
+    """
+    return ProjectProperties(build_project_definition(**definition), {})
```

Comment:
why is the context empty? This will break all rendering in sql or files

Reply:
But we don`t need to render anything - we're just converting `snowflake.yml`